service.md

```diff
@@ -56,24 +106,51 @@ Connect to the VirtualMachine by using a node IP and node port outside the clust
 $ ssh cirros@$NODE_IP -p 30000
 ```
 
-### Expose VirtualMachine as a single port service
+### Expose VirtualMachine as a LoadBalancer service
 
-Create a single port `service` object by using `kubectl`:
+Expose the RDP port (3389) of a VirtualMachine running on KubeVirt by creating `LoadBalancer service`. Here is an example of a LoadBalancer service:
 
-```bash
-$ kubectl expose pod virt-launcher-testvm-ephemeral-9bqv4 --port=27017 --target-port=22 --name=vmservice
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: windows2012r2
+  name: lbsvc
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - nodePort: 31830
```

Comment:
the nodeport ( and the port) is not needed in this case

Reply:
Removed nodeport in yaml file